resolwe/flow/management/commands/prepare_runtime.py

```diff
@@ -16,6 +16,59 @@
 from resolwe.storage import settings as storage_settings
 
 
+def copy_dir(source: Path, destination: Path):
+    """Copy dir from source to destination.
+
+    The shutil copytree method could be used but it will not overwrite
+    directories (option added in Python 3.8, while we support Python 3.6).
+
+    This method iterates through all entries in source directory and copies
+    them over to the destination directory (overwriting them if they
+    already exist).
+    """
+    for source_entry in source.rglob("*"):
+        relative_source_path = source_entry.relative_to(source)
+        destination_entry = destination / relative_source_path
+        if source_entry.is_dir():
+            destination_entry.mkdir(exist_ok=True, parents=True)
+        elif source_entry.is_file() or source_entry.is_symlink():
+            destination_entry.parent.mkdir(exist_ok=True, parents=True)
+            shutil.copy(source_entry, destination_entry)
+
+
+def prepare_python_runtime(
+    destination_dir: Path, runtime_subdir: str = "python_runtime"
+):
+    """Prepare python runtime in the given folder.
+
+    Python runtime will be copied to a subfolder runtime_subdir inside
+    destination_dir. The destination_dir must be a directory.
+
+    :raises AssertionError: when destination_dir is not a directory or source
+        path of the Python process runtime can not be determined.
+    """
+    assert (
+        destination_dir.is_dir()
+    ), f"The path '{destination_dir}' must be a directory."
+
+    for runtime_class_name in settings.FLOW_PROCESSES_RUNTIMES:
+        module_name, class_name = runtime_class_name.rsplit(".", 1)
+        runtime_module = import_module(module_name)
```

Comment:
I don't like that all modules are imported here, as they are configurable, and arbitrary code can be executed.

Can you do it with the following snippet?
```py
import importlib
spec = importlib.util.find_spec(module_name)
spec.origin
```

Reply:
Yes, that works like a charm. Thanks.